cmd/kube-proxy/app/server.go

```diff
@@ -509,8 +509,9 @@ with the apiserver API to configure the proxy.`,
 	opts.AddFlags(cmd.Flags())
 
 	// TODO handle error
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
-
+	if err := cmd.MarkFlagFilename("config", "yaml", "yml", "json"); err != nil {
```

Comment:
Can you detail why this particular call may fail?  Fatal-ing may be reasonable, but we should explain why it is reasonable.  A code-ref link in a comment above would be great.